deploy/deploy_analytics_terminal.yml

```diff
@@ -6,7 +6,7 @@
     - {role: puppet_disable, when: "disable_puppet"}
     - {role: epel, when: "use_epel and not 'production' in group_names"}
     - {role: iptables, when: "iptables_config"}
-    - {role: common, when: "install_glusterfs"}
+#    - {role: common, when: "install_glusterfs"}
```

Comment:
@olivermel Please make sure not to include commented out lines in PR's, as if I missed this it could break many things.  If you need to disable the common role, for example, use the EXTRA_VARS environment variable like so:
```
EXTRA_VARS='{ install_glusterfs: "False" }' vagrant up...
```